internal/v4/api.go

```diff
@@ -95,7 +95,7 @@ func New(store *charmstore.Store, config charmstore.ServerParams) *Handler {
 			"id-revision":   h.entityHandler(h.metaIdRevision, "_id"),
 			"id-series":     h.entityHandler(h.metaIdSeries, "_id"),
 			"manifest":      h.entityHandler(h.metaManifest, "blobname"),
-			"perm":          h.baseEntityHandler(h.metaPerm, "acls"),
+			"perm":          h.puttableBaseEntityHandler(h.metaPerm, h.putMetaPerm, "acls"),
```

Comment:
Right now it is admin auth only, I thought. Maybe this was the idea of admin access in the doc?